### ISSUE TITLE
Language Boot 

### ISSUE THREAD

**Issue by Bygos** [nminchow/black-book#7]
Hello can u thing about more language to your boot ?
I from Poland ( Polish language ) for example.

**Comment by Bygos** [comment]
> 

Boot Diablo4 Black book !

**Comment by nminchow** [comment]
Yeah! I just pushed support for localization. Would you be willing to help with polish translation? https://github.com/nminchow/black-book/blob/main/src/i18n/LOCALIZATION.md
